### ISSUE TITLE
Tabbing does not work to searchable multiselects

### ISSUE THREAD

**Issue by enkrs** [shentao/vue-multiselect#685]
The recent "Improve the `singleLabel` slot feature" commit - [Multiselect.vue line 35](https://github.com/shentao/vue-multiselect/commit/13f7fb6617e580443543996f0e0d8c4467d25652#diff-f0fe4bcbb4c9df5319d483db4c27ee58R35) introduces a regression for issues #411 , #515 and similar. With this shift-tabbing and even tabbing to searchable multiselects does not work anymore.

This is because `v-show` on `<input` sets `style="display: none"` on the DOM element. Such elements ignore tabindex in the browser.

In earlier commits "[Set tabindex on input instead of container if searchable](https://github.com/shentao/vue-multiselect/commit/3e963b21b75e76894b6d72ce12832668cbb98eb2#diff-f0fe4bcbb4c9df5319d483db4c27ee58L254)"  I can see that there have been attempts to avoid "display: none".

I've made this fiddle to reproduce it. Pushing Tab on the keyboard from first input should jump to multiselect. This is working with 2.0.8 libs, but not 2.1.0 libs:

https://jsfiddle.net/rjr8n2oc/1/

**Comment by shentao** [comment]
Thanks for reporting! 
This is a serious problem. :/ With the current event architecture it’s quite easy to break such things – that’s why the 3.0 is so needed. 
Will try to fix this issue during the weekend. If someone can do this sooner, I will gladly accept the PR.

**Comment by leovanhaaren** [comment]
Has this been fixed already? I'm facing the same issue but don't have time to investigate and/or make a PR for it. :(

**Comment by leovanhaaren** [comment]
Ah I see that the fix for it has been merged into master but a new release hasn't been created yet. Is it possible to release an package update to npm @shentao ?

It's this commit that I'm after;
https://github.com/shentao/vue-multiselect/commit/65b989501aae3b7b29ef441271392929f7065480

**Comment by adam-lynch** [comment]
The test suite failed to run on CircleCI for that commit https://circleci.com/gh/shentao/vue-multiselect/360?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link. Maybe that's why it wasn't published?

**Comment by shentao** [comment]
Sadly most people who provide PRs do not add the required tests (nor run them). I merged some of the PRs today but haven’t yet fixed the tests. Once this is done I will be able to make a release to npm. :(

**Comment by leovanhaaren** [comment]
@shentao Did you already find time to do this or is there anything you need help with?

**Comment by leovanhaaren** [comment]
@shentao see above ^

**Comment by ScriptMint** [comment]
+1

**Comment by jclausen** [comment]
@shentao :  Any word on when this bug fix will be released?

**Comment by TomFranssen** [comment]
I would also like to know when this will be released.

**Comment by johndatserakis** [comment]
Standing by for this - definitely a much needed feature.

**Comment by ricardobanegas** [comment]
I downgraded to `"vue-multiselect": "2.0.3"` and everything works beautifully.

**Comment by jclausen** [comment]
@ricardobanegas : That may be a final option - to lock in to 2.0.3 - if it looks like this won't be addressed in the near future. I would prefer to be able to take advantage of some of the tweaks and fixes in the 2.1.0 release, if at all possible.  It's a great component so I'm not complaining. If that's what we need to do, long term, so be it.

**Comment by derekphilipau** [comment]
@ricardobanegas Just downgraded to 2.0.3 and it seems to be working fine.  Thanks for the tip.

**Comment by DM2489** [comment]
Definitely still required.

**Comment by wujekbogdan** [comment]
Is there any workaround for this issue? I've just downgraded Vue Multiselect to 2.0.8  and it seems to work fine, but I can't find any changelog, so I don't know what can possibly break because of the downgrade.

**Comment by AndreaRavano** [comment]
+1

**Comment by rickyruiz** [comment]
**Workaround:**

Create a `computed` property that returns an object which overrides the `tabindex` attribute.

```
computed: {
  override() {
    return {
     tabIndex: 0,
    }
  },
},
```

Then use `v-bind`:

```
<multiselect
  v-model="value"
  :options="options"
  v-bind="override"
 />
```

Demo:
https://jsfiddle.net/rickyruizm/uc5sfL7e/

**Comment by wujekbogdan** [comment]
@rickyruiz
Thanks for the workaround, but unfortunately it's not perfect. It works only if you move forward with tab, but it doesn't work if you try to go backwards with shift+tab.

**Comment by samoldenburg** [comment]
Any update on this? Serious negative effects on user experience here.

**Comment by rickyruiz** [comment]
@wujekbogdan With 2.1.2 there is no need for a workaround anymore, but I am not sure if it fixes `shit+tab`, I think it does not.

**Comment by wujekbogdan** [comment]
> I think it does not

It does, but not always, I didn't find the rule yet. It seems to have something to do with the value of the input.

**Comment by serakfalcon** [comment]
Guys, the issue is on line 3 of https://github.com/shentao/vue-multiselect/blob/master/src/Multiselect.vue
` :tabindex="searchable ? -1 : tabindex"`
IF searchable tabindex is -1. Not sure why this is a good idea. Should be 
`:tabindex="tabindex"`
or if you want default to be -1 if searchable
`:tabindex="(searchable && tabindex !== undefined) ? -1 : tabindex"`

**Comment by UnknownBlackspider** [comment]
Same issue here it didn't work for me :'(
